util/config/fixpath.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import os
+from sys import argv
+
+
+def splitescaped(s, delim=":", escape='\\'):
```

Comment:
Just when I thought I had a clever solution....  I switched to this approach in latest commit.